samples/Calculator/tests/Calculator.Console.Tests/Calculator.Console.Tests.csproj

```diff
@@ -0,0 +1,21 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net7.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+
+    <IsPackable>false</IsPackable>
+    <IsTestProject>true</IsTestProject>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.7.1" />
+    <PackageReference Include="xunit" Version="2.5.0" />
```

Comment:
xunit?!?!?! 😄 we should use MSTest or nUnit...

Reply:
Server.IntegrationTests are referecning Nunit. Lets use one thing everywhere.

Reply:
I don't recall with method parallism mstest won't parallelize, but fine if not